Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './Header.css';
+import FooterItem from './FooterItem'
+
+const FooterContainer = () => (
+    <div clasName='footer-container'>
+        <FooterItem src='#' num='7' />
```

Comment:
Try to remember to pass these as props and keep your functional components pure so that you can use them again.